doc/cephfs/createfs.rst

```diff
@@ -48,7 +48,7 @@ Once the pools are created, you may enable the file system using the ``fs new``
 
 .. code:: bash
 
-    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover]
+    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [<fscid:int>] [--recover] [<set>...] [--yes-i-really-really-mean-it]
```

Comment:
```suggestion
    $ ceph fs new <fs_name> <metadata> <data> [--force] [--allow-dangerous-metadata-overlay] [--yes-i-really-really-mean-it] [--fscid <fscid:int>] [--recover] [--yes-i-really-really-mean-it] ["set" <setting> <value> ...] 
```

Ah, `[--yes-i-really-really-mean-it]` is now an option. Make sure you have tests for that.

Reply:
`--yes-i-really-really-mean-it` is exclusive only to `set`, that's why i put it after set.

Reply:
It's unconventional to have switches follow positional arguments.

Reply:
noted. will make the changes.